logstash-core/spec/logstash/shutdown_watcher_spec.rb

```diff
@@ -1,12 +1,12 @@
 # encoding: utf-8
 require "spec_helper"
-require "logstash/shutdown_controller"
+require "logstash/shutdown_watcher"
 
-describe LogStash::ShutdownController do
+describe LogStash::ShutdownWatcher do
 
   let(:check_every) { 0.01 }
   let(:check_threshold) { 100 }
-  subject { LogStash::ShutdownController.new(pipeline, check_every) }
+  subject { LogStash::ShutdownWatcher.new(pipeline, check_every) }
```

Comment:
you can use `described_class`  here so the only change would have been the describe block